README.md

```diff
@@ -1,119 +1,105 @@
+2. Add/install Microsoft's "[Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh)" extension.
+
+#### To complete along with your interviewer:
+1. Create ssh config file *if one does not already exist* (if `.ssh` **DOES** already exist, proceed to Step 2).
+	1. Navigate to root directory: `cd ~`
+  2. In root directory (~), `mkdir .ssh`
+  3. `ls .ssh`
+  4. Inside `~/.ssh`, `touch config`
+2. Your interviewer will send you some connection information to paste into this newly created config file. Open it up, paste the connection info, then save. Example:
   ```Host hc-interview
     HostName EC2_INSTANCE_URL_GOES_HERE
     User candidate
     LocalForward 3000 localhost:3000
     LocalForward 3001 localhost:3001```
-- Open VS Code
-- Command + Shift + P and enter `Remote-SSH: Connect to Host`
-- Select `hc-interview`
-- A new VS Code tab will open and ask you to enter a password in the dropdown. The password is `candidate`
-- Once entered, select `Open Folder` from the left side of the screen
-- Select `/home/candidate/interview-internal-apps-2022/` from the dropdown
-- You will be asked to enter the password again. `candidate`
-- You should now be able to see the files and code base we will be working in on the left side of the screen. Any code changes you make and save, your interviewers will be able to see. 
-
-# Windows Setup:
-**TODO BEFORE THE INTERVIEW:**
-
-- Install [VS code](https://code.visualstudio.com/Download)
-- Add/install Microsoft's "[Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh)" extension
-
-**TODO WITH YOUR INTERVIEWER:**
-
-- Create ssh config file *if one does not already exist*
-	- Navigate to root directory 'C:\Users\Username'. `cd ~`
-    If `.ssh` **DOES NOT** exist:
-    - In root directory (~), `mkdir .ssh` 
-    - `ls .ssh`
-    - Inside `~/.ssh`, `touch config`
-	If `.ssh` **DOES** already exist, proceed to next steps:
-- Open config file
-- Paste connection info into the `config` file and save config file. Your interviewer will send you the connection info, but see example below:
+3. Open VS Code, then Command + Shift + P and enter `Remote-SSH: Connect to Host`
+4. Select `hc-interview`. A new VS Code tab will open and ask you to enter a password in the dropdown. The password is `candidate`.
+5. Select `Open Folder` from the left side of the screen.
+6. Select `/home/candidate/interview-internal-apps-2022/` from the dropdown. You will be asked to enter the password again: `candidate`
+7. You should now be able to see the files of the codebase on the left side of the screen. Any code changes you make and save, your interviewers will be able to see.
+
+### Windows Setup
+
+#### To complete before the interview:
+1. Install [VS code](https://code.visualstudio.com/Download).
+2. Add/install Microsoft's "[Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh)" extension.
```

Comment:
These instructions are the same whether you're on PC or Mac. Do we want to consolidate them, or leave as-is?

Reply:
I think leave them as is. Just so folks can follow only one guide depending on their os.